app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -33,12 +33,25 @@ def update
       end
     end
 
+    def self.opted_in_to_remote(idv_session:)
+      if IdentityConfig.store.in_person_proofing_opt_in_enabled &&
+         IdentityConfig.store.in_person_proofing_enabled &&
+         idv_session.service_provider&.in_person_proofing_enabled
+        idv_session.skip_doc_auth == false
+      else
```

Comment:
i don't quite understand the else statement. It's setting `skip_doc_auth` to nil or to false?

Reply:
So this section is checking if the user can access the `hybrid_handoff_controller`. We have to support a couple of different cases:
1. If the service provider has opt in enabled, we only want to allow users to access the hybrid handoff page from the `how_to_verify` page if the `skip_doc_auth` value is false. This prevents users from skipping ahead because that value is initially nil.
2. If the opt in is not enabled, we need to support both nil and false. This also doesn't prevent users from skipping ahead, but they can't skip because hybrid handoff is shown as the default if opt in is not enabled.